cms/djangoapps/models/settings/course_grading.py

```diff
@@ -184,6 +192,13 @@ def update_section_grader_type(descriptor, grader_type, user):
             del descriptor.graded
```

Comment:
(this really belongs a few lines up)
If the only valid values of `grader_type` are "graded" and "ungraded", we should be explicit about it in the if/else conditions, i.e.
```python
if grader_type == 'graded':
    descriptor.format = grader_type
    descriptor.graded = True
elif grader_type == 'notgraded':
    del descriptor.format
    del descriptor.graded
else:
    return # or raise
```